src/common/Makefile.am

```diff
@@ -86,11 +87,6 @@ libcommon_internal_la_SOURCES += \
 	common/pick_address.cc
 endif
 
-if LINUX
```

Comment:
@trociny agreed . see http://www.opensolarisforum.org/man/man2/uname.html @rohanmars could you help confirm this?